R/filter_pd.R

```diff
@@ -0,0 +1,229 @@
+#' Filter up to First PD Date
+#'
+#' Filter a dataset to only include the source parameter records up to and
+#' including the first PD. These records are passed to downstream derivations
+#' regarding responses such as BOR.
```

Comment:
Updated.

I think this should be a general rule in our documentation. By the way, is this the standard way of introducing/resolving acronyms? In the admiral documentation I found three ways:
- "Adds a record for BMI/Body Mass Index using"
- "Adds a record for BSA (Body Surface Area) using"
- "Adds a record for mean arterial pressure (MAP) for"

Reply:
@bundfussr  agree - i'm not too worried about having a single convention though (as would be difficult to enforce) just as long as we have acronyms explained somewhere on the page, so that we don't assume too much prior knowledge for users to be able to adopt